.golangci.yaml

```diff
@@ -0,0 +1,14 @@
+linters:
+  enable:
+    - gosec
```

Comment:
We have to enable `revive` (replacement for deprecated `golint`) and `mispell` as they are disabled by default and we were using those previously in our `sanity-check`.

Ref: https://golangci-lint.run/usage/linters/#disabled-by-default-linters--e--enable

Reply:
Thanks, @shahulsonhal. Replaced `golint` with `revive`.